jenkins_job_wrecker/modules/triggers.py

```diff
@@ -256,12 +256,65 @@ def ghprbtrigger(top, parent):
             ghpr['permit-all'] = get_bool(child.text)
         elif child.tag == 'autoCloseFailedPullRequests':
             ghpr['auto-close-on-fail'] = get_bool(child.text)
+        elif child.tag == 'blackListCommitAuthor':
+            ghpr['black-list-commit-author'] = child.text
```

Comment:
IMO `black-list-commit-author` should also be a list, similarly to `black-list-labels`

Reply:
Well, you may be right, but the input on jenkins UI does not allow for new lines, so I will split by spaces.